src/libertem/analysis/masks.py

```diff
@@ -41,6 +41,75 @@ def get_preset_dtype(self):
         return self.parameters.get('dtype', None)
 
 
+class MasksResultSet(AnalysisResultSet):
+    """
+    Running a :class:`MasksAnalysis` vial :meth:`libertem.api.Context.run` on a dataset
+    returns an instance of this class.
+
+    If any of the masks or the dataset contain complex numbers, the regular mask results
+    attributes carry the absolute value of the results, and additional attributes with real
+    part, imaginary part, phase and full complex result are available.
+
+    .. versionadded:: 0.3.0.dev0
+
+    Attributes
+    ----------
+    mask_0, mask_1, ..., mask_<n> : libertem.analysis.base.AnalysisResult
```

Comment:
Maybe also document `__getitem__`: `result[0] == mask_0` etc. - maybe it should also be augmented by an iteration interface for looping over mask results, something like this (in the base class):

```
def __iter__(self):
    return iter(self.results)
```

I would be fine with doing this in the next iteration of the docs.

Reply:
Done! Implementing the iterator method was the least of the issue...